### PR TITLE
Add Contributing.md

### PR DESCRIPTION
**Pull request by dfed**
cc @alanf @justinseanmartin

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+###Sign the CLA
+
+All contributors to your PR must sign our [Individual Contributor License Agreement (CLA)](https://spreadsheets.google.com/spreadsheet/viewform?formkey=dDViT2xzUHAwRkI3X3k5Z0lQM091OGc6MQ&ndplr=1). The CLA is a short form that ensures that you are eligible to contribute.
```
Comment (alanf) [inline Contributing.md]: Can we clarify that folks only need to do this once for any Square project?

Something like:

```
You only need to do this once, so if you've done this for another Facebook open source project, you're good to go. If you are submitting a pull request for the first time, just let us know that you have completed the CLA and we can cross-check with your GitHub username.

File: Contributing.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Keep your Pull Requests small. Small PRs are easier to reason about which makes them significantly more likely to get merged.
+
+###Issues before features
+
+If you want to add a feature, please file an [Issue](issues) first. An Issue gives us the opportunity to discuss the requirements and implications of a feature with you before you start writing code.
+
+###Backwards compatibility
+
+Respect the minimum deployment target. If you are adding code that uses new APIs, make sure to prevent older clients from crashing or misbehaving. Our CI runs against our minimum deployment targets, so you will not get a green build unless your code is backwards compatible.
```
Comment (alanf) [inline Contributing.md]: Deployment targets aren't a part of Space Commander

Comment (dfed) [inline Contributing.md]: Fair point! Should we get rid of the entire `Backwards compatibility` section then?

Comment (alanf) [inline Contributing.md]: Backwards compatibility is still something that matters for folks who have previously run `setup-repo.sh`, so I wouldn't cut it entirely. Maybe just the section about targets.

Comment (dfed) [inline Contributing.md]: The entire section is about targets...

File: Contributing.md (hunk 1 above)
Comment (dfed) [inline Contributing.md]: done

File: Contributing.md (hunk 2 above)
Comment (alanf) [inline Contributing.md]: Existing installs of Space Commander should continue to work after upgrading to a version that includes your changes. Some changes may need to include a migration plan.

Comment (dfed) [inline Contributing.md]: replaced!

**Comment by alanf** [comment]
LGTM

**Pull request merged**